operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -362,25 +362,22 @@ metadata:
     categories: Streaming & Messaging
     certified: 'false'
     containerImage: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0-0
-    createdAt: 2024-03-09 21:27:00
+    createdAt: 2024-08-22 15:19:00
     description: Streams for Apache Kafka is a massively scalable, distributed, and high performance data streaming platform based on Apache Kafka®.
     repository: https://github.com/strimzi/strimzi-kafka-operator
     support: Red Hat
-    features.operators.openshift.io/disconnected: "true"
-    features.operators.openshift.io/proxy-aware: "true"
-    features.operators.openshift.io/fips-compliant: "true"
-    features.operators.openshift.io/cnf: "false"
-    features.operators.openshift.io/cni: "false"
-    features.operators.openshift.io/csi: "false"
-    features.operators.openshift.io/tls-profiles: "false"
-    features.operators.openshift.io/token-auth-aws: "false"
-    features.operators.openshift.io/token-auth-azure: "false"
-    features.operators.openshift.io/token-auth-gcp: "false"
-    operators.openshift.io/valid-subscription: |-
-      ["Red Hat AMQ", "Red Hat Application Foundations"]
+    operators.openshift.io/infrastructure-features: |-
+      ["disconnected", "proxy", "proxy-aware", "fips"]
     operators.operatorframework.io/internal-objects: |-
       ["strimzipodsets.core.strimzi.io"]
     olm.skipRange: '>=2.7.0-0 <2.8.0-0'
+    features.operators.openshift.io/disconnected: “true”
+    features.operators.openshift.io/fips-compliant: “true”
+    features.operators.openshift.io/proxy-aware: “true”
+    features.operators.openshift.io/tls-profiles: “false”
+    features.operators.openshift.io/token-auth-aws: “false”
+    features.operators.openshift.io/token-auth-azure: “false”
+    features.operators.openshift.io/token-auth-gcp: “false”
```

Comment:
why this section was moved down and we missed some of the the fields?

Reply:
Moved back and missing fields returned